sdk/iptables/iptables.go

```diff
@@ -100,6 +106,16 @@ func Setup(cfg Config) error {
 		// Redirects outbound TCP traffic hitting PROXY_REDIRECT chain to Envoy's outbound listener port.
 		cfg.IptablesProvider.AddRule("iptables", "-t", "nat", "-A", ProxyOutputRedirectChain, "-p", "tcp", "-j", "REDIRECT", "--to-port", strconv.Itoa(cfg.ProxyOutboundPort))
 
+		if cfg.ConsulDNSIP != "" {
```

Comment:
Add a comment that we want to apply dns rules before the rule that directs all TCP traffic, so that the traffic going to port 53 goes through this rule first.